pkg/kubelet/rocket/rocket.go

```diff
@@ -0,0 +1,452 @@
+/*
+Copyright 2015 CoreOS Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rocket
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+	"syscall"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubelet/volume"
+	"github.com/coreos/go-systemd/dbus"
+	"github.com/coreos/go-systemd/unit"
+	"github.com/golang/glog"
+)
+
+const (
+	kubernetesUnitPrefix = "K8S"
+	systemdServiceDir    = "/etc/systemd/system"
+
+	unitKubernetesSection = "X-Kubernetes"
+	rocketIDKey           = "RocketID"
+	unitPodName           = "POD"
+)
+
+const (
+	rocketBinName = "rkt"
+
+	podStateEmbryo    = "embryo"
```

Comment:
yes, @jonboulle but it's not exported last time I looked... Should add a TODO here, thx!